src/Index/page.less

```diff
@@ -39,20 +39,3 @@
   max-height: 40vh;
   object-fit: contain;
 }
-
-.index-page .ant-carousel .slick-dots-bottom {
```

Comment:
why removing these? these are intended for the index page that other env has

Reply:
which environment? (so i can test) as they seem to be not needed

Reply:
im seeing the problem ill look into fixing it everywhere

Reply:
NCT has this at least, there was 1 env that has carousel but I can't remember now 🤷‍♂️ 

FYI there is [this commit](https://github.com/uc-cdis/data-portal/pull/803/commits/40f38376d56b6567f9024e920f761165a98c19d6) that was for the discovery page

Gloabally overwriting antd styles might be causing some problems on other pages, so I'd suggest you get in touch with @mpingram , or put a scope on style overwritings